PROTOCOL.md

```diff
@@ -1174,15 +1174,15 @@ Example:
 
 This is the API for clients to get a table version without any other extra information. The server usually can implement this API effectively. If a client caches information about a shared table locally, it can store the table version and use this cheap API to quickly check whether their cache is stale and they should re-fetch the data.
 
-Note: We are migrating this method from HEAD to GET, and with a `/version` suffix. Please use the new API as we'll deprecate the support of HEAD API soon. 
+Note: This method is migrating from HEAD to GET, and with a `/version` suffix. Please use the new API as the support of HEAD API will be deprecated on the server side 07/01/2023. 
```

Comment:
on the server side  -> on the server side on ?

Reply:
updated.